maltd-frontend/cypress/integration/tests/find-users.js

```diff
@@ -0,0 +1,39 @@
+/* global cy, before */
+// / <reference types="Cypress" />
+
+import LandingPage from "../../support/pageObjects/landing-page";
+
+describe("The landing Page", () => {
+  before(() => {
+    // Runs once before all tests in the block
+    cy.fixture("userData").then(data => {
+      this.data = data;
+    });
+  });
+
+  it("Find users", () => {
+    const landingPage = new LandingPage();
+    // Launches the url
+    cy.visit("/");
+
+    // Validates the Find button is disabled by default
+    landingPage.getFindButton().should("be.disabled");
+
+    // Validating invalid inputs and error message
+    let i = this.data.userName.invalid.length;
+
+    while (i > 0) {
+      i -= 1;
```

Comment:
question: if we are decrementing first thing in the loop, can we just change the conditional in the loop or what the first value we check is?